node/dataSources/catalog.ts

```diff
@@ -153,9 +154,15 @@ export class CatalogDataSource extends IODataSource {
     orderBy = '',
     from = 0,
     to = 9,
-    map = ''
+    map = '',
+    filterUnavailableBySegment = false,
```

Comment:
```suggestion
    filterUnavailableBySegment,
```

I think GraphQL will auto assign this to false, but even if it didn't, it would be falsy by default

Reply:
but this is on the dataSource